lib/ASTGen/Sources/ASTGen/Literals.swift

```diff
@@ -1,47 +1,52 @@
 import CASTBridging
-import SwiftParser
 import SwiftSyntax
 
 extension ASTGenVisitor {
   public func visit(_ node: StringLiteralExprSyntax) -> ASTNode {
-    let loc = bridgedSourceLoc(for: node)
+    let openDelimiterOrQuoteLoc = self.bridgedSourceLoc(for: node.openDelimiter ?? node.openQuote)
```

Comment:
I know it was like this before, but I'd appreciate a TODO/FIXME to handle string interpolations.